crates/ruma-events/src/enums.rs

```diff
@@ -31,6 +31,8 @@ event_enum! {
         "m.push_rules" => super::push_rules,
         "m.secret_storage.default_key" => super::secret_storage::default_key,
         "m.secret_storage.key.*" => super::secret_storage::key,
+        #[cfg(feature = "unstable-msc4278")]
+        "m.media_preview_config" => super::media_preview_config,
```

Comment:
Is it on purpose that this is not using the unstable prefix?

Reply:
You're absolutely right it should use:
`io.element.msc4278.media_preview_config`

I will refactor it immediately.